integration/src/wallets/metamask.ts

```diff
@@ -83,23 +87,26 @@ export function selfTest(get: () => core.HDWallet): void {
 
   it("uses correct eth bip44 paths", () => {
     if (!wallet) return;
-    [0, 1, 3, 27].forEach((account) => {
+    // MM doesn't support multi-account
+    const accountNumbers = metamask.isMetaMask(wallet) ? [0] : [0, 1, 3, 27];
+    if (metamask.isMetaMask(wallet)) return;
+    accountNumbers.forEach((account) => {
       const paths = wallet.ethGetAccountPaths({
-        coin: "Ethereum",
+        coin: "ethereum",
```

Comment:
revert, pref to use `coin` as it appears in the `slip44Table`. can `toLowerCase` the `describePath` switch statement in `shapeshift-multichain` or just update to coin keys (uppercase)

Reply:
df02ebb